React/CoreModules/RCTDeviceInfo.mm

```diff
@@ -201,17 +214,20 @@ - (void)interfaceFrameDidChange
 
 - (void)_interfaceFrameDidChange
 {
+  UIApplication *application = RCTSharedApplication();
```

Comment:
could you also remove this variable that is unused? It's make our sanity checks exploding!

Reply:
done